### PR TITLE
fix(api): fix generated openapi spec

### PR DESCRIPTION
**Pull request by Intuinewin**
Hi, first thanks to @bmanifold for his awesome work! I've not yet tested the API but here is a first PR fixing various small mistakes in the generated openapi spec:

- Schema names cannot contain spaces
- Add missing path parameters in the spec
- Remove duplicated endpoint for creating an identity (not sure about that, I'll let you check)

If you want to validate the generated spec you can paste it here: https://editor.swagger.io/ (or at the bottow of your swagger ui)

Please review commit by commit

### CODE REVIEW

**Comment by jamilbk** [comment]
Thanks for the contribution @Intuinewin!

@bmanifold What I've been doing for these contrib PRs to get them to run CI is:

- Do an initial review
- Once it looks good change the base to `contrib` which tracks `main`
- Merge this PR to `contrib`
- Open a PR to merge `contrib` to `main`

That'll run CI and still maintain the author's commits so that they're properly attributed.

File: elixir/apps/api/lib/api/router.ex [hunk 1]
```diff
@@ -52,7 +52,6 @@ defmodule API.Router do
 
     resources "/actors", ActorController, except: [:new, :edit] do
       resources "/identities", IdentityController, except: [:new, :edit, :update]
-      post "/providers/:provider_id/identities/", IdentityController, :create
```
Comment (bmanifold) [inline elixir/apps/api/lib/api/router.ex:55]: Ah, this was actually my mistake.  I made this change toward the end and forgot to come back to the router and make the proper change.  The route being deleted here is actually intended and the `create` route created by the `resources` function was meant to be removed.  So, this should actually be:
```
resources "/identities", IdentityController, except: [:new, :edit, :create, :update]
post "/providers/:provider_id/identities/", IdentityController, :create
```

**Review by bmanifold — COMMENTED** [review]
@Intuinewin, thank you very much for the contribution!  It's really appreciated.  Overall the PR looks pretty good, just had one small comment.

File: elixir/apps/api/lib/api/router.ex (hunk 1 above)
Comment (Intuinewin) [reply elixir/apps/api/lib/api/router.ex:55]: Ok, that was the part I wasn't sure about. I updated the code and tests, it should be good now

**Comment by bmanifold** [comment]
@Intuinewin, thanks again!  I just merged your PR in to our `contrib` branch and have opened up another PR to allow all of the tests to run.  If you'd like to keep track of it, the PR is #6008

**Pull request merged**